### DISCUSSION TITLE
What is needed to make this into main HA?

### DISCUSSION THREAD

**Discussion by sanchosk** [syssi/homeassistant-goecharger-mqtt#95] [General]
This integration is amazing.
What needs to be done to convert this from HACS into main HA repository?
Can I help somehow?

**Comment by syssi** [comment]
Good question. I'm not sure about it. In general a library is required between the device and the Home Assistant integration but I don't know this applies to MQTT too. If this abstraction isn't required it's probably just missing test coverage of the config flow.
